lib/searchyll/indexer.rb

```diff
@@ -1,5 +1,6 @@
 require 'json'
 require 'net/http'
+require 'yaml'
```

Comment:
Jekyll already uses SafeYAML - should we just use that? `SafeYAML.load(content)`

Reply:
@drusellers Unluckly i don't know how SafeYAML works but if this doesn't akter the parsed object's structure it is still fine.